### ISSUE TITLE
Cannot install gammapy using pip

### ISSUE THREAD

**Issue by MaxNoe** [gammapy/gammapy#1205]
```
pip install gammapy
```
 fails with a really strange recursion error

```
Collecting gammapy
  Using cached gammapy-0.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-yknxpn97/gammapy/setup.py", line 61, in <module>
        cmdclassd = register_commands(PACKAGENAME, VERSION, RELEASE)
      File "/tmp/pip-build-yknxpn97/gammapy/.eggs/astropy_helpers-1.0.7-py3.6.egg/astropy_helpers/setup_helpers.py", line 281, in register_commands
        from .commands.build_sphinx import AstropyBuildSphinx, AstropyBuildDocs
      File "/tmp/pip-build-yknxpn97/gammapy/.eggs/astropy_helpers-1.0.7-py3.6.egg/astropy_helpers/commands/build_sphinx.py", line 14, in <module>
        from sphinx.setup_command import BuildDoc as SphinxBuildDoc
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/sphinx/setup_command.py", line 23, in <module>
        from sphinx.application import Sphinx
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/sphinx/application.py", line 33, in <module>
        from sphinx.environment import BuildEnvironment
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/sphinx/environment/__init__.py", line 45, in <module>
        from sphinx.util.websupport import is_commentable
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/sphinx/util/websupport.py", line 11, in <module>
        from sphinxcontrib.websupport.utils import is_commentable  # NOQA
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/sphinxcontrib/websupport/__init__.py", line 15, in <module>
        from sphinxcontrib.websupport.core import WebSupport  # NOQA
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/sphinxcontrib/websupport/core.py", line 17, in <module>
        from jinja2 import Environment, FileSystemLoader
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/jinja2/__init__.py", line 33, in <module>
        from jinja2.environment import Environment, Template
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/jinja2/environment.py", line 15, in <module>
        from jinja2 import nodes
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/jinja2/nodes.py", line 19, in <module>
        from jinja2.utils import Markup
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/site-packages/jinja2/utils.py", line 486, in <module>
        MutableMapping.register(LRUCache)
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/abc.py", line 158, in register
```
Lots of lines here
```
py", line 207, in __subclasscheck__
        ok = cls.__subclasshook__(subclass)
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/typing.py", line 884, in __extrahook__
        if issubclass(subclass, scls):
      File "/home/maxnoe/.local/anaconda3/lib/python3.6/typing.py", line 1153, in __subclasscheck__
        return super().__subclasscheck__(cls)
    RecursionError: maximum recursion depth exceeded while calling a Python object
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-yknxpn97/gammapy/
```

**Comment by cdeil** [comment]
@MaxNoe - this is a bug that appeared and was fixed a few months ago, after the v0.6 release.

We have a handful of things to do and will release v0.7 very soon:
https://github.com/gammapy/gammapy/milestone/10

For now, please install the dev version of Gammapy (`git clone` and `pip install .`)

We'll make stable releases for frequently going forward.

Sorry for the trouble!

**Comment by MaxNoe** [comment]
This can be done in one line:
```pip install https://github.com/gammapy/gammapy/archive/master.tar.gz```
